components/atomic/Flag/patterns/EarlyAccess/styles.js

```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components";
+import { fluidScale, tokens } from "@/styles/globalStyles";
+import Flag from "@/atomic/Flag";
+
+export const EarlyAccess = styled(Flag)`
```

Comment:
`Flag` is in charge of its shape, `EarlyAccess` is in charge of how stuff looks and lays out inside that shape.